### PR TITLE
ceaser decoder

### PR DESCRIPTION
**Pull request by TreverArnold**

### CODE REVIEW

File: javaScript/README.md [hunk 1]
```diff
@@ -12,3 +12,7 @@
   * Module: `code-challenges/javaScript/arr.diff.js`
   * Tests: `code-challenges/javaScript/arr.diff.test.js`
   * URL: [challenge url](https://www.codewars.com/kata/523f5d21c841566fde000009)
+### Break Caeser (5kyu)
+  * Module: `code-challenges/javaScript/break.ceaser.js`
+  * Tests: `code-challenges/javaScript/break.ceaser.test.js`
+  * URL: [challenge url](https://www.codewars.com/kata/523f5d21c841566fde000009)
```
Comment (W-Ely) [inline javaScript/README.md:18]: this doesn't link to the correct Challenge.

**Review by W-Ely — CHANGES_REQUESTED** [review]
The file name `javaScript/break.ceaser.js` should be `javaScript/breakCeaser.js`. I probably missed this on other files.  lowerCamelCase on Javascript files that don't represent or define React Components. 

The exception is the test file is `theName.test.js`

Since I can't see the challenge it's hard to say about the WORD being needed just for tests.  It has a code smell, but perhaps you have to do it that way, would need to see the challenge.

**Comment by TreverArnold** [comment]
> Since I can't see the challenge it's hard to say about the WORD being needed just for tests. It has a code smell, but perhaps you have to do it that way, would need to see the challenge.

I was more so talking about my selection of words, I had attempted to import a list of the top 1000 english words, but ran into issues

**Comment by W-Ely** [comment]
> I was more so talking about my selection of words

Yup, I see from the challenge that you need this global, a-okay.

**Review by W-Ely — APPROVED** [review]

**Pull request merged**